### PR TITLE
Fix Blogpost ZAP SSRF setup

### PR DESCRIPTION
**Pull request by DiogoMRSilva**
Planned release date: March 9th

Signed-off-by: Diogo Silva diogo.m.r.silva@tecnico.ulisboa.pt

### CODE REVIEW

File: site/content/blog/2020-03-09-zap-ssrf-setup/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ If you want to have a valid HTTPS certificate you can run some reverse proxy on
 ssh -N -R 9999:localhost:38193 myUser@something.mydomain.org
 ```
 
-__Troubleshot__:
+### Troubleshot:
```
Comment (thc202) [inline site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: It's missing an o, `Troubleshoot`.

Comment (kingthorin) [inline site/content/blog/2020-03-09-zap-ssrf-setup/index.md]:
```suggestion
### Troubleshoot:
```

**Comment by thc202** [comment]
Thanks!

File: site/content/blog/2020-03-09-zap-ssrf-setup/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ If you want to have a valid HTTPS certificate you can run some reverse proxy on
 ssh -N -R 9999:localhost:38193 myUser@something.mydomain.org
 ```
 
-__Troubleshot__:
+### Troubleshoot:
 - open the ports on the firewall
 - make shure you allow port forwarding on the server ssh config `/etc/ssh/sshd_config`:
```
Comment (thc202) [inline site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: `sure`, just noticed while reading in the site.

Comment (DiogoMRSilva) [reply site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: Sorry, I did a new PR

Comment (thc202) [reply site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: No worries.

**Pull request merged**